src/communication/network_interface_sr.c

```diff
@@ -10736,8 +10746,15 @@ flashback_verify_time (THREAD_ENTRY * thread_p, time_t start_time, time_t end_ti
 
 	  if (ret_time >= end_time)
 	    {
+	      char start_date[20];
+	      char db_creation_date[20];
+
+	      strftime (start_date, 20, "%d-%m-%Y:%H:%M:%S", localtime (&start_time));
+	      strftime (db_creation_date, 20, "%d-%m-%Y:%H:%M:%S", localtime (&log_Gl.hdr.db_creation));
+
 	      /* out of range : start_time (ret_time) can not be greater than end_time */
-	      /* TODO : er_set() */
+	      er_set (ER_NOTIFICATION_SEVERITY, ARG_FILE_LINE, ER_FLASHBACK_INVALID_TIME, 3, start_date,
+		      db_creation_date, start_date);
```

Comment:
`Invalid time (start_date) to start flashback. Time is required to be set between db_creation_date) and (start_date).` is correct? It seems that we need `end_date`

Reply:
This is the part I was worried about whether I should put the end date. 
However, in this situation 
1. start date  < end date 
2, there is no log value between the start date and the end date
3. write an error message asking user to enter a value before the start date.